### PR TITLE
Added initialization SQL and Shell scripts and PostgREST container

### PR DESCRIPTION
**Pull request by AfandiM**
The command to restore the database from the dump was put in a shell initialization script, so now the database will automatically be restored from the dump file (no need for step 13)

I also added an initialization SQL script to add a test schema and table for testing purposes and also add a web_anon role that will allow anonymous users to view (read) some tables

To see the PostgREST API, you can open up localhost:3000/<table_name> and you will see a JSON format of the data in the table.

In the code I wrote, I specified, in line 31 of the compose file, `public` and `test_schema` as the schemas exposed by the PostgREST API, but because of the way PostgREST containers work, only tables in the first schema name specified (in this case `public`) can be accessed through localhost:3000/<table_name> alone. To access the tables of the other schema (`test_schema`), you will need to specify the `Accept-Profile` header with a value of `test_schema` in the GET request. An example is shown below with Postman

image

References issue #13

### CODE REVIEW

File: compose.yml [hunk 1]
```diff
@@ -5,11 +5,13 @@ services:
     environment:
       - POSTGRES_USER=${POSTGRES_USER}
       - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
-      - POSTGRES_DB=uinnovate_test_db # Default database's name
+      - POSTGRES_DB=${POSTGRES_DB_NAME} # Default database's name
```
Comment (WillTrem) [inline compose.yml:8]: Would you be able to add the POSTGRES_DB_NAME to the .env.template please? :)

**Review by WillTrem — COMMENTED** [review]
Looks super good!! Would you be able to update the readme.md file so that it reflects the changes you made? It would be super helpful for the rest of the team!

File: compose.yml (hunk 1 above)
Comment (AfandiM) [reply compose.yml:8]: Completely forgot. Done!

**Comment by AfandiM** [comment]
> Looks super good!! Would you be able to update the readme.md file so that it reflects the changes you made? It would be super helpful for the rest of the team!

Done! You can check to see if its explanatory enough

**Review by WillTrem — DISMISSED** [review]
Tested the changes on my end, works great! Approved :)

**Review by SamRfk — APPROVED** [review]
Looks like there aren't any conflicts with main anymore. Everything looks good, more files will be added to structure the database in compose.yml

**Pull request merged**